Tools/SmokeTests/QueueDriverConnectionSmokeTest.php

```diff
@@ -4,6 +4,7 @@
 
 use Smartbox\CoreBundle\Utils\SmokeTest\Output\SmokeTestOutput;
 use Smartbox\CoreBundle\Utils\SmokeTest\SmokeTestInterface;
+use Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers\AsyncQueueDriverInterface;
```

Comment:
Is it needed this use if there is no change in the class itself?

Reply:
You,re right. This use is some rest of code once we were testing the new interface. I removed it now.